src/wasm-binary-reader.c

```diff
@@ -701,6 +701,12 @@ static WasmResult logging_on_table(uint32_t index,
   FORWARD(on_table, index, elem_type, elem_limits);
 }
 
+static WasmResult logging_on_memory_count(uint32_t count, void* user_data) {
```

Comment:
hehe, just noticed this myself. But you can use the convenience macros instead of writing this manually (see LOGGING_UINT32 above).